fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -28,9 +28,9 @@ def self.run(params)
 
         # Set version if it is latest
         if version == 'latest'
-          # Try to grab the live version first, else fallback to edit version
-          latest_version = app.live_version(platform: platform) || app.edit_version(platform: platform)
-          version = latest_version.version
+          # Try to grab the edit version first, else fallback to live version
+          latest_version = app.edit_version(platform: platform) || app.live_version(platform: platform)
+          version = nil
```

Comment:
Is there a reason for setting this to nil?

Reply:
Yes, the reason is in my commit message:

> nil out version since AppVersion.version will not necessarily match any version in a BuildTrain

Reply:
Just in case that's not clear, if I understand correctly, AppVersion.version corresponds with the version number set in iTunes Connect, which is user settable and does not have to match the version number in the IPA.  For example, for necessary but irrelevant reasons, our versions are 6.12 and 1006.12.